.gitignore

```diff
@@ -1,8 +1,8 @@
 __pycache__
 *.pyc
 *.pyo
+.*
```

Comment:
👎 
We shouldn't ignore all the files beginning from a dot.

Reply:
you can forcefully added to index the necessary files.

Reply:
I can do it with any other file, why not to ignore *.* then? I guess explicitly ignored should be only the files which are not meant to be appeared in the repo, such as temporary files, build results and other autogenerated files. Files beginning from dot are not such kind of files.